deploy/stacks/pipeline.py

```diff
@@ -936,21 +951,13 @@ def set_albfront_stage(self, target_env, repository_name):
                     role=self.expanded_codebuild_role.without_policy_updates(),
                     vpc=self.vpc,
                 ),
-            ],
-            post=self.evaluate_post_albfront_stage(target_env)
+            ]
         )
+        if target_env.get('custom_auth') is None:
+            albfront_stage.add_post(self.cognito_config_action(target_env))
 
-    def evaluate_post_albfront_stage(self, target_env):
-        if target_env.get("enable_cw_rum", False):
-            post=[
-                self.cognito_config_action(target_env),
-                self.cw_rum_config_action(target_env),
-            ]
-        else:
-            post=[
-                self.cognito_config_action(target_env),
-            ]
-        return post
+        if target_env.get('enable_cw_rum', False):
```

Comment:
Similar to above let's restrict based on both enable cw rum and custom auth is NOT present